tests/e2e/framework/common.go

```diff
@@ -2211,3 +2221,68 @@ func (f *Framework) WaitForGenericRemediationToBeAutoApplied(remName, remNamespa
 	}
 	return nil
 }
+
+func (f *Framework) AssertComplianceSuiteIsSuspended(name string) error {
```

Comment:
We should be ok to remove this, and replace it with a similar function that asserts a binding is suspended.